demos/time_series_handling.ipynb

```diff
@@ -99,7 +99,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "ts.psd_df(signal_df, NFFT = 512).plot(loglog=True, ylabel=\"PSD\", title=\"PSD of time series\")"
+    "ts.psd_df(signal_df, nfft = 512).plot(loglog=True, ylabel=\"PSD\", title=\"PSD of time series\")"
```

Comment:
Why isn't there a unit test, that needs this adjustment. Do we have a hole in the unit tests here?